android/modules/locale/src/java/ti/modules/titanium/locale/LocaleModule.java

```diff
@@ -33,6 +39,62 @@ public LocaleModule()
 		super("Locale");
 	}
 
+	@Kroll.setProperty
+	public void setApplicationLocales(String locales)
+	{
+		if (Build.VERSION.SDK_INT < 33) {
+			Log.w(TAG, "This property is only supported on Android API level 33 and above.");
+			return;
+		}
+		LocaleListCompat appLocale = LocaleListCompat.forLanguageTags(locales);
+		AppCompatDelegate.setApplicationLocales(appLocale);
+	}
+
+	@Kroll.getProperty
+	public KrollDict[] getApplicationLocales()
```

Comment:
It is very uncommon to have different types for the getter and setter. And there is `Ti.Locale.setLanguage` already. So I would propose:
- Use the new Android 33+ API in `Ti.Locale.setLanguage(language)`
- Expose the getter as a new Titanium API (if there isn't one already to get language details - if so, update that one)

Reply:
Seems like this PR is with no use so I'll close it